### PR TITLE
MDS-3645 Proponent can update start and stop work dates

### PR DESCRIPTION
**Pull request by JustinMacaulay**
Adds a section to the MineSpace overview that allows the proponent to update the start and stop work dates for their mine.

### CODE REVIEW

File: services/minespace-web/src/components/dashboard/mine/overview/MineWorkInformation.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                        <Row gutter={16}>
+                          <Col span={24} />
+                        </Row>
+                      </div>
+                    )}
+                  </Row>
+                  <Row>
+                    {info && (
+                      <Col span={22}>
+                        {/* NOTE: Ant Design has no easy way to right-align "Descriptions" so plain HTML tags are used here. */}
```
Comment (exzizt) [inline services/minespace-web/src/components/dashboard/mine/overview/MineWorkInformation.js:190]: Good comment

**Review by exzizt — APPROVED** [review]
Nice work!
giphy

File: services/minespace-web/src/components/Forms/AddMineWorkInformationForm.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        <Col span={24}>
+          <Form.Item>
+            <Field
+              id="work_comments"
+              name="work_comments"
+              label="Comments"
+              placeholder="Enter Comments"
+              minRows={3}
+              disabled={!props.isEditMode}
+              component={renderConfig.AUTO_SIZE_FIELD}
```
Comment (mmumby) [inline services/minespace-web/src/components/Forms/AddMineWorkInformationForm.js:140]: Is the form even visible if not in EditMode?

Comment (JustinMacaulay) [reply services/minespace-web/src/components/Forms/AddMineWorkInformationForm.js:140]: I dont think it renders but I also dont see the harm in this for now

**Pull request merged**